src/components/panel/panel.js

```diff
@@ -1067,25 +1068,52 @@ function MdPanelPosition() {
 
   /** @private {string} */
   this._right = '';
+
+  /** @private {string} */
+  this._xPosition = '';
+
+  /** @private {string} */
+  this._yPosition = '';
 }
 
 
 /**
- * Sets absolute positioning for the panel.
- * @return {!MdPanelPosition}
+ * Possible values of xPosition.
+ * @enum {string}
  */
-MdPanelPosition.prototype.absolute = function() {
-  this._absolute = true;
-  return this;
+MdPanelPosition.xPosition = {
+  CENTER: 'center',
+  ALIGN_LEFT: 'align-left',
+  ALIGN_RIGHT: 'align-right',
+  ALIGN_START: 'align-start',
+  ALIGN_END: 'align-end',
+  OFFSET_LEFT: 'offset-left',
+  OFFSET_RIGHT: 'offset-right',
+  OFFSET_START: 'offset-start',
+  OFFSET_END: 'offset-end'
+};
```

Comment:
Why have the items that are `left` and `right` at all as opposed to just `start` and `end` ?